### ISSUE TITLE
Template Strings

### ISSUE THREAD

**Issue by k15a** [styled-components/elm-styled#6]
Elm currently doesn't have template strings which is the reason I decided to implement each CSS declaration as a function. Maybe a way to support both would be great so it's faster to get into elm-styled. The problem is that inserting values into a String is pretty ugly in elm currently.

```Elm
fontSize = 2

header = styledCss div
    """
        font-size: """ ++ (toString fontSize) ++ """rem;
        font-color: red;
    """
```

Maybe it's not worth to support that because elm-css is accepted very well in the elm community as well. In addition with this method you would lose the power of typed Css.

**Comment by drager** [comment]
I agree with that maybe it isn't worth it. And as you say, elm-css is a fine alternative. I like the approach of implementing each CSS declaration as a function, so maybe this package should: "Do One Thing and Do It Well."
